src/ucp/core/ucp_request.inl

```diff
@@ -318,6 +319,12 @@ ucp_request_try_send(ucp_request_t *req, unsigned pending_flags)
 static UCS_F_ALWAYS_INLINE void
 ucp_request_send(ucp_request_t *req, unsigned pending_flags)
 {
+    VALGRIND_CHECK_MEM_IS_DEFINED(&req->send.state.uct_comp.func,
+                                  sizeof(req->send.state.uct_comp.func));
+    VALGRIND_CHECK_MEM_IS_DEFINED(&req->send.state.uct_comp.count,
+                                  sizeof(req->send.state.uct_comp.count));
+    VALGRIND_CHECK_MEM_IS_DEFINED(&req->send.state.uct_comp.status,
+                                  sizeof(req->send.state.uct_comp.status));
```

Comment:
maybe do one check for `sizeof(req->send.state.uct_comp)`?

Reply:
it failed due to padding on uct_comp.status